src/main/java/com/programmers/springbootbasic/exception/GlobalThymeExceptionHandler.java

```diff
@@ -0,0 +1,27 @@
+package com.programmers.springbootbasic.exception;
+
+import com.programmers.springbootbasic.exception.exceptionClass.CustomException;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class GlobalThymeExceptionHandler {
+
```

Comment:
```suggestion
    @ResponseStatus(value = HttpStatus.BAD_REQUEST)
```
GlobalThymeExceptionHandler가 처리하는 예외들은 별도의 응답 코드가 나가지만 HttpStatus는 200으로 보여지는데 의도하신 내용일까요?